.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was not sure if something from .idea folder should be checked-in. If it is not, you can remove all existing files from git.

Reply:
Sure, I will clean these files.

Reply:
You put this file in `.gitignore` list. Should it be deleted?

Reply:
I think it should not be deleted, since it is ignored, anyone could maintain his sdk locally, and will not be commit. I'm sorry that it is hard to download these sdk in China.